lightning/src/ln/channelmanager.rs

```diff
@@ -10357,9 +10357,21 @@ where
 	#[cfg(c_bindings)]
 	create_refund_builder!(self, RefundMaybeWithDerivedMetadataBuilder);
 
+	/// Retrieve our cached [`Offer`]s for receiving async payments as an often-offline recipient.
```

Comment:
Maybe add some explanation on who is going to call the API? Somehow I had to think that if we're offline, we don't need an API?